### ISSUE TITLE
findSubmitter behavior

### ISSUE THREAD

**Issue by josh** [javan/form-request-submit-polyfill#1]
Hey @javan,

Was curious about the `findSumitter` flow and how it aligns to the spec. It seems that if no submitter is given, a default one is chosen by:

https://github.com/javan/form-request-submit-polyfill/blob/555a974cb55047f9cba4a14d162c4f51d2c73dc1/form-request-submit-polyfill.js#L8-L9

It seems like it will lead to incorrect parameters being passed along.

```html
<form id="form">
  <input type=submit name=wrong-target value=Submit>
</form>

<script>
  document.getElementById("form").requestSubmit()
</script>
```

The polyfill will end up passing along `?wrong-target=Submit` where the correct behavior is to pass nothing.

##

CC: @muan

**Comment by javan** [comment]
You're right. I think we can just skip that whole branch of the condition:
```diff
--- a/form-request-submit-polyfill.js
+++ b/form-request-submit-polyfill.js
@@ -5,8 +5,6 @@
     if (submitter) {
       validateSubmitter(submitter, this)
       submitter.click()
-    } else if (submitter = findSubmitter(this)) {
-      submitter.click()
     } else {
       submitter = document.createElement("input")
       submitter.type = "submit"
@@ -26,15 +24,4 @@
   function raise(errorConstructor, message, name) {
     throw new errorConstructor("Failed to execute 'requestSubmit' on 'HTMLFormElement': " + message + ".", name)
   }
-
-  function findSubmitter(form) {
-    var inputs = form.elements
-    var index = inputs.length
-    while (index--) {
-      var input = inputs[index]
-      if (input.type == "submit" && !input.disabled) {
-        return input
-      }
-    }
-  }
 })(HTMLFormElement.prototype);
```

**Comment by josh** [comment]
haha, I was about to just delete the lines in a PR. But thought I might be missing something

**Comment by javan** [comment]
Nope! This was a quick extraction from an app's helper function, and a total oversight. Please do open a PR to claim your open source points. :)

**Comment by javan** [comment]
Thank you. Just published v2.0.0.
